armory/__init__.py

```diff
@@ -6,6 +6,11 @@
 logger = logging.getLogger(__name__)
 logger.addHandler(logging.NullHandler())
 
+
+# Semantic Version
+__version__ = "0.13.0"
```

Comment:
Since `armory.docker.images` references the semantic version, I wanted this to proceed before importing.